samples/nrf9160/nrf_cloud_mqtt_multi_service/README.rst

```diff
@@ -229,6 +267,37 @@ If you are using a different board or build target, or would like to use a custo
 
 Enable :kconfig:option:`CONFIG_MODEM_ANTENNA_GNSS_EXTERNAL` to use an external antenna.
 
+.. _nrf_cloud_mqtt_multi_service_customizing_LED_status_indication:
+
+Customizing LED status indication
+=================================
+
+To disable LED status indication after a connection to nrf Cloud has been established at least once, disable :ref:`CONFIG_LED_VERBOSE_INDICATION <CONFIG_LED_VERBOSE_INDICATION>`.
+
+To make the LED turn off while the sample is idle rather than show an idle pattern, disable :ref:`CONFIG_LED_CONTINUOUS_INDICATION <CONFIG_LED_CONTINUOUS_INDICATION>`.
+
+
+.. _nrf_cloud_mqtt_multi_service_led_third_party:
+
+Configuring LED status indication for third-party boards
+========================================================
+
+This sample assumes that the board being targeted either has a single RGB LED with PWM support, or four discrete LEDs available.
+
+For third-party boards, you may select the RGB LED option by enabling both the :ref:`CONFIG_LED_INDICATION_PWM <CONFIG_LED_INDICATION_PWM>` and :ref:`CONFIG_LED_INDICATION_PWM <CONFIG_LED_INDICATOR_RGB>` options.
+In this case, the board must have a devicetree entry marked as compatible with the `Zephyr pwm-leds`_ driver.
+
+Otherwise, the four-LED option (:ref:`CONFIG_LEDINDICATION_GPIO <CONFIG_LED_INDICATION_GPIO>` and :ref:`CONFIG_LED_INDICATOR_4LED`) will be enabled by default as long as there is a devicetree entry compatible with the `Zephyr gpio-leds`_ driver.
```

Comment:
```suggestion
Otherwise, the four-LED option (:ref:`CONFIG_LEDINDICATION_GPIO <CONFIG_LED_INDICATION_GPIO>` and :ref:`CONFIG_LED_INDICATOR_4LED <CONFIG_LED_INDICATOR_4LED>`) is enabled by default as long as there is a devicetree entry compatible with the `Zephyr gpio-leds`_ driver.
```